code/src/microphone_controller.cpp

```diff
@@ -0,0 +1,40 @@
+#include <microphone_controller.hpp>
+
+namespace r2d2::microphone {
```

Comment:
Could you run clang format on this file?

Reply:
Ran clang format on this file!